client/client.go

```diff
@@ -118,6 +118,7 @@ type (
 		//  - Get(ctx context.Context, valuePtr interface{}) error: which will fill the workflow
 		//    execution result to valuePtr, if workflow execution is a success, or return corresponding
 		//    error. This is a blocking API.
+		// If workflow not found, the Get() will timeout.
```

Comment:
More generic, the service side API doesn't know if the workflow is found. Basically, it will wait for the workflow to close and return the closed event. So this API will timeout if the workflow does not closed in time.